homeassistant/const.py

```diff
@@ -16,7 +16,7 @@
 REQUIRED_NEXT_PYTHON_HA_RELEASE: Final = ""
 
 # Format for platform files
-PLATFORM_FORMAT: Final = "{platform}.{domain}"
+PLATFORM_FORMAT: Final = "{domain}.{platform}"
```

Comment:
It's confusing that this is inverted now. When we talk about platforms, the domain means the domain that entities of the platform are grouped by, eg, an entity_id `sensor.test_id` has the domain `sensor` and belongs to a platform that has the domain `sensor` by an integration `test_integration`. We should stay consistent with those semantics here too.

Reply:
This looks incorrect:
https://github.com/home-assistant/core/blob/e03ccb5ab624327031011adf3c82575ee597eced/homeassistant/components/tts/__init__.py#L547-L549

Reply:
I wonder if we should remove the `PLATFORM_FORMAT` constant? It's only used in two places.

Reply:
As discussed on discord, we should probably use `PLATFORM_FORMAT` in more places instead. Let's do that in a follow-up 👍